deepmd/infer/model_devi.py

```diff
@@ -42,17 +48,29 @@ def calc_model_devi_f(
         minimum deviation of force in all atoms
     avg_devi_f : numpy.ndarray
         average deviation of force in all atoms
+    fs_devi : numpy.ndarray
+        deviation of force in all atoms
```

Comment:
This is a breaking change, we may want to add an option `atomic: bool=False`  to control the output of atomic force deviation.